patches/vite-plugin-twig-drupal@1.3.0.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/src/index.js b/src/index.js
```

Comment:
Very cool! 

I thought about this for a long time last week and decided that it wasn't worth trying to bundle the JS together together with the Twig function that gets exported from the vite plugin. In the first place, Wingsuit doesn't do it, you have to explicitly import the JS for sub-components and in the second, it's unexpected, you wouldn't assume that a plugin for twig components would go looking for JS files too.

But I really like this implementation, it's straightforward and makes for nice story files.

I especially like using a the schema to check the validity of pattern files, that's a great touch.

In terms of patching the original vite-twig plugin, that's cool for now, but I think we should either:

1. Make a PR to support this (may be as an option) to the current library
2. Fork it and publish separately